bundles/org.openhab.core/src/main/java/org/openhab/core/storage/Storage.java

```diff
@@ -21,8 +21,8 @@
 import org.eclipse.jdt.annotation.Nullable;
 
 /**
- * A Storage is the generic way to store key-value pairs in ESH. Each Storage
- * implementation can store its data differently, e.g in-memory or in-database.
+ * A {@link Storage} is the generic way to store key-value pairs in ESH. Each storage implementation can store its data
```

Comment:
There's also an ESH reference waiting to be updated here. :-)

Reply:
Cleaned it up.